### PR TITLE
[Validator] Updated Lithuanian translations

### LINKED ISSUE #53771: Missing translations for Lithuanian (lt)
Hello,

There are some translation work needed for Lithuanian (lt) and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Lithuanian (lt) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf)

<details>
<summary>Show related strings</summary>

#### src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 37 | This is not a valid IP address. | Ši vertė nėra galiojantis IP adresas. | Needs review |
| 51 | No temporary folder was configured in php.ini. | php.ini nesukonfigūruotas laikinas aplankas, arba sukonfigūruotas aplankas neegzistuoja. | Needs review |
| 59 | This is not a valid International Bank Account Number (IBAN). | Ši vertė nėra galiojantis Tarptautinis Banko Sąskaitos Numeris (IBAN). | Needs review |
| 81 | This is not a valid Business Identifier Code (BIC). | Ši vertė nėra galiojantis Verslo Identifikavimo Kodas (BIC). | Needs review |
| 83 | This is not a valid UUID. | Ši vertė nėra galiojantis UUID. | Needs review |
| 111 | The detected character encoding is invalid ({{ detected }}). Allowed encodings are {{ encodings }}. | Nustatyta simbolių koduotė yra netinkama ({{ detected }}). Leidžiamos koduotės yra {{ encodings }}. | Needs review |
| 112 | This value is not a valid MAC address. | Ši vertė nėra galiojantis MAC adresas. | Needs review |
| 113 | This URL is missing a top-level domain. | Šiam URL trūksta aukščiausio lygio domeno. | Needs review |

</details>

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by LinasRam**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #53771
| License       | MIT

Reviewed and updated Lithuanian translations with "needs-review-translation" state.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf [hunk 1]
```diff
@@ -432,15 +432,15 @@
...
             </trans-unit>
             <trans-unit id="112">
                 <source>This value is not a valid MAC address.</source>
-                <target state="needs-review-translation">Ši vertė nėra galiojantis MAC adresas.</target>
+                <target>Ši reikšmė nėra tinkamas MAC adresas.</target>
             </trans-unit>
             <trans-unit id="113">
                 <source>This URL is missing a top-level domain.</source>
-                <target state="needs-review-translation">Šiam URL trūksta aukščiausio lygio domeno.</target>
+                <target>Šiam URL trūksta aukščiausio lygio domeno.</target>
```
Comment (norkunas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf:443]: Maybe "Šioje nuorodoje trūksta aukščiausio lygio domeno." ?

Comment (LinasRam) [reply src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf:443]: I don't think so, URL and "link" are not exactly the same thing. It's usually not translated in other languages, too.

Comment (norkunas) [reply src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf:443]: Yeah, but in lt URL could mean nothing for non-dev people :thinking: ok leave as is :)

**Comment by nicolas-grekas** [comment]
Thank you @LinasRam.

**Pull request merged**